Assets/Mirror/Weaver/Weaver.cs

```diff
@@ -179,7 +179,6 @@ class Weaver
         public static MethodReference sendEventInternal;
         #endregion
 
-        public static AssemblyDefinition scriptDef;
```

Comment:
Where did it go?   it looks like CurrentAssembly is not declared anywhere in this PR

Reply:
307612c